rpcserver.go

```diff
@@ -3499,3 +3613,25 @@ func (r *rpcServer) RemoveUTXOLease(ctx context.Context,
 
 	return &wrpc.RemoveUTXOLeaseResponse{}, nil
 }
+
+// MarshalAssetFederationSyncConfig returns an RPC ready asset specific
+// federation sync config.
+func MarshalAssetFederationSyncConfig(
```

Comment:
Should we call this `MarshalAssetFedSyncCfg` to shorten?

Reply:
Also, shouldn't this be in `taprpc/marshal.go`?

Reply:
> shouldn't this be in taprpc/marshal.go?

It makes use of a struct from the `universe` package which will be a import cycle in the `taprpc` package:
```
	imports github.com/lightninglabs/taproot-assets/taprpc
	imports github.com/lightninglabs/taproot-assets/taprpc/universerpc
	imports github.com/lightninglabs/taproot-assets/taprpc: import cycle not allowed
```

This is because the universe proof courier imports `universerpc` into the `universe` package. That probably needs to be disentangled.